roles/nginx/README.md

```diff
@@ -0,0 +1,52 @@
+# mirsg.infrastructure.nginx
+
+This role is for configuring [nginx](https://www.nginx.com/) as a
+[reverse proxy](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/)
+on CentOS 7 or RockyLinux 9.
+
+## Role Variables
+
+| Name                  | Description                                                                                                                                                                                     |
+| --------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `nginx_owner`         | The OS user that will have ownership of the nginx service file and directory. Defaults to `root`                                                                                                |
+| `nginx_group`         | The OS group that will have ownership of the nginx service file and directory. Defaults to `root`                                                                                               |
+| `nginx_log_folder`    | The path to the nginx logs. Defaults to `/var/log/nginx`                                                                                                                                        |
+| `nginx_http_port`     | The port to listen on for HTTP connections. Defaults to `80`                                                                                                                                    |
+| `nginx_https_port`    | The port to listen on for HTTPS connections. Defaults to `80`                                                                                                                                   |
+| `nginx_proxy_port`    | The port to forward requests to. Defaults to `8080` (tomcat)                                                                                                                                    |
+| `nginx_root`          | The path to search for static files. Defaults to `/usr/share/tomcat/webapps/ROOT`                                                                                                               |
+| `nginx_conf_template` | The template to use for generating the NGINX config. See currently available [templates](templates/). Defaults to `nginx_xnat.j2`, which is used to configure NGINX as a reverse proxy for XNAT |
```

Comment:
Again I'm not sure about having an XNAT default. Can we have a basic reverse proxy config?